webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx

```diff
@@ -409,7 +410,7 @@ const TableActionBar: React.FC<Props> = ({
               Compare
             </Button>
           )}
-        </Columns>
+        </Row>
       </Column>
       {batchAction && (
```

Comment:
nit: now that the modal components have wrapper divs they're being treated as children of the row, causing the extra padding on the far right of the action bar. not a huge deal, but it would be nice to fix.

Reply:
Something I don't quite understand is why the modal is inserted into the tree here? Since modals are portaled why don't we just tack them onto the end of components?